packages/aws-cdk-lib/aws-stepfunctions/lib/states/state.ts

```diff
@@ -76,8 +82,67 @@ export interface StateProps {
    * @default - None
    */
   readonly resultSelector?: { [key: string]: any };
+
+  /**
+   * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
+   *
+   * @see
+   * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html#input-output-parameters
+   *
+   * @default No parameters
+   */
+  readonly parameters?: { [name: string]: any };
```

Comment:
For my understanding, why does this need to be moved from `JsonPathCommonOptions` to `JsonPathStateOptions`?

Reply:
`Parameters` and some properties are not common options so moved from `JsonPathCommonOptions` that using by majority states. 
 See more details, please refer the document.
- [Common state fields in workflows](https://docs.aws.amazon.com/step-functions/latest/dg/statemachine-structure.html#amazon-states-language-common-fields)